tensorboard/components/tf_card_heading/tf-card-heading-style.ts

```diff
@@ -46,5 +42,7 @@
         flex-grow: 0;
       }
     </style>
-  </template>
-</dom-module>
+  </custom-style>
+`;
+template.setAttribute('style', 'display: none;');
```

Comment:
Just curious, is this needed?  I thought templates are already not-rendered?

Reply:
I got inspired by https://github.com/PolymerElements/iron-flex-layout/blob/master/iron-flex-layout.js#L429. 

Actually, you are correct though because this template where we apply this style is not used for anything since we are only taking its content. Removed.